src/js/TopicController.js

```diff
@@ -7,7 +7,7 @@ goog.provide('ga_topic_controller');
       function($scope, gaGlobalOptions) {
           $scope.options = {
             defaultTopicId: gaGlobalOptions.defaultTopicId,
-            thumbnailUrlTemplate: gaGlobalOptions.resourceUrl + 'img/{Topic}.jpg',
+            thumbnailUrlTemplate: gaGlobalOptions.resourceUrl + 'img/topics.png',
```

Comment:
Remove this line, useless with the sprite css.

And look for others place we use this variable `thumbnailUrlTemplate` then remove it.